src/myr/Myr.js

```diff
@@ -11,6 +11,10 @@ class Myr {
         this.assets = [];
         this.res = { els: this.els, assets: this.assets };
         this.sceneEl = document.querySelector("a-scene");
+        this.seed = 0; 
+        this.randCounter = 0;
+        this.oldRandomCounter = 0;
+        this.oldSeed = 0;
```

Comment:
Similarly to this.cursor, it would be good to have these as a JSON object.

Reply:
changed to being a JSON object